packages/cfd/src/Containers/cfd-compare-accounts/cfd-instruments-label-highlighted.tsx

```diff
@@ -17,19 +15,14 @@ const CFDInstrumentsLabelHighlighted = ({ trading_platforms }: TCompareAccountsC
             {is_eu_user ? (
                 <React.Fragment>
                     {iconData
-                        .filter(item => item.highlighted)
+                        .filter(instrument => instrument.is_available !== false)
```

Comment:
[M]
```suggestion
                        .filter(instrument => instrument.is_available)
```

Reply:
`is_available: selected_region === REGION.NON_EU` is used to filter out instruments that are not available in the EU. Other instruments don't have the `is_available` field, so we need to explicitly check for `false`.